camel/agents/chat_agent.py

```diff
@@ -134,24 +137,31 @@ def __init__(
 
         self.model_config: BaseConfig
         if self.model_type == ModelType.GPT_4_TURBO_VISION:
-            if model_config is not None and not isinstance(
-                model_config, ChatGPTVisionConfig
-            ):
+            assert model_config is None or isinstance(
+                model_config, ChatGPTConfig
+            )
+            # Now OpenAI GPT-4 vision model requires these params
+            # to be specified in the config, raise error if they
+            # are not specified:
```

Comment:
from the openai doc it seems that `stop` and `max_tokens` are not required for `gpt-4-turbo`
refer to: https://platform.openai.com/docs/guides/vision

Reply:
I met an error before because of the default value `None` for `stop` and `max_tokens` in the `ChatGPTConfig`. Changed to OpenAI's default `NOT_GIVEN` resolves the error.